central/graphql/resolvers/images.go

```diff
@@ -48,6 +48,7 @@ type ImageResolver interface {
 	Signature(ctx context.Context) (*imageSignatureResolver, error)
 	SignatureVerificationData(ctx context.Context) (*imageSignatureVerificationDataResolver, error)
 	TopCvss(ctx context.Context) float64
+	BaseImageInfo(ctx context.Context) ([]*baseImageInfoResolver, error)
```

Comment:
Is this the only resolver change needed for `images` and `images_v2` table?
cc: @c-du  @jvdm  @ksurabhi91  @charmik-redhat

Reply:
I guess no. We need to tell at image component level if it belong to base or belong to add-on part